saleor/graphql/product/mutations/attributes.py

```diff
@@ -515,6 +555,130 @@ def perform_mutation(cls, _root, info, **data):
         return cls(product_type=product_type)
 
 
+class PageAttributeAssign(BaseMutation):
```

Comment:
I think these mutations should be moved somewhere to `graphql/page` module if they are related to pages.

Reply:
Yes, it might be a better place for it. But anyway mutation for creating page Attribute is in this module. IMO, we should move all attribute related stuff to a separate model in the future.